eox_nelp/course_experience/api/v1/tests/mixins_helpers.py

```diff
@@ -35,6 +40,31 @@ def setUp(self):  # pylint: disable=invalid-name
         self.my_course, _ = CourseOverview.objects.get_or_create(id=BASE_COURSE_ID)
         self.client.force_authenticate(self.user)
 
+    def make_relationships_data(self):
+        """
+        Make the relationships dict with custom extra attributes based in the attributes of the serializers
+        withe variables COURSE_OVERVIEW_EXTRA_ATTRIBUTES and USER_EXTRA_ATTRIBUTES
+
+        Returns:
+            dict: relationships dict with the corresponding shape, and key-values.
+        """
+        return {
+            "author": {
+                "data": {
+                    "type": "User",
+                    "id": f"{self.user.id}",
+                    "attributes": map_attributes_from_instance_to_dict(self.user, USER_EXTRA_FIELD_MAPPING),
```

Comment:
why are not you using the method get_course_extra_attributes and get_user_extra_attributes

Reply:
I passed, maybe I was more focused on the map_dictionary function. Is better what you suggest.
[de7a82f](https://github.com/eduNEXT/eox-nelp/pull/64/commits/de7a82f3a1b778ba2f0bc671fc55cfbb9e1a72c0)